WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartTaskDetails.kt

```diff
@@ -0,0 +1,95 @@
+package org.wordpress.android.ui.quickstart
+
+import org.wordpress.android.R.drawable
+import org.wordpress.android.R.string
+import org.wordpress.android.fluxc.store.QuickStartStore
+import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTask
+
+/**
+ * Static data that represents user facing description of quick start tasks.
+ */
+enum class QuickStartTaskDetails(
+    val taskString: String,
+    val titleResId: Int,
+    val subtitleResId: Int,
+    val iconResId: Int
+) {
```

Comment:
@ashiagr Same here, Thanks for converting this file to kotlin, Do you think we can add a resource annotation(@StringRes,@DrawableRes) improvement? 🤔